addons/website_profile/views/website_profile.xml

```diff
@@ -125,7 +125,7 @@
                 </div>
                 <div class="col-9 mb-3">
                     <div class="card">
-                        <div class="card-body">
+                        <div class="card-body o_profile_editor">
```

Comment:
Rename the class in something specific to the application:
`o_wprofile_editor_form` (see the convention for the second part: 'wprofile' = the name of the application, shortened)